From 8a8506320acf3ea18940a6e3a447d7d394a3bd8e Mon Sep 17 00:00:00 2001
From: David Gouarin <david.gouarin@thalesgroup.com>
Date: Mon, 12 Apr 2021 11:05:54 -0500
Subject: [PATCH] jlibtool cross with host CC

jlibtool.c: while cross compiling, use host CC instead of target CC to build jlibtool

Upstream: Not applicable

Signed-off-by: David Gouarin <david.gouarin@thalesgroup.com>
Signed-off-by: Matt Weber <matthew.weber@rockwellcollins.com>
[Fabrice: update for 3.2.3]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Dario: make the patch to be applied with fuzz factor 0]
Signed-off-by: Dario Binacchi <dario.binacchi@amarulasolutions.com>
[Waldemar: update for 3.2.6]
Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
---
 scripts/libtool.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/libtool.mk b/scripts/libtool.mk
index 64c2fe0804..9dc7910463 100644
--- a/scripts/libtool.mk
+++ b/scripts/libtool.mk
@@ -40,8 +40,8 @@ all install: ${JLIBTOOL}
 # binary!
 ${JLIBTOOL}: ${top_makedir}/jlibtool.c
 	$(Q)mkdir -p $(dir $@)
-	$(Q)echo CC jlibtool.c
-	$(Q)${CC} $< -o $@ ${JLIBTOOL_DEFS}
+	$(Q)echo HOSTCC jlibtool.c
+	$(Q)${HOSTCC} $< -o $@ ${JLIBTOOL_DEFS}
 
 clean: jlibtool_clean
 
